mods/boats/init.lua

```diff
@@ -9,10 +9,12 @@ end
 
 
 local function get_sign(i)
-	if i == 0 then
-		return 0
+	if i > 0 then
+		return 1
+	elseif i < 0 then
+		return -1
 	else
-		return i / math.abs(i)
+		return 0
 	end
 end
```

Comment:
Changing something here seems a bit out of scope.

Btw. there's also a `math.sign` from builtin which does the same:
https://github.com/minetest/minetest/blob/42e1a127140965cac1be6e51e48192e341c2a29e/doc/lua_api.txt#L2437-L2441

Ergo `local get_sign = math.sign` would be enough.